nativelib/src/main/resources/gc/immix/Block.c

```diff
@@ -110,20 +114,16 @@ void Block_Recycle(Allocator *allocator, BlockHeader *blockHeader) {
 
 void Block_Print(BlockHeader *block) {
```

Comment:
I have an impression this method is not used anywhere. If that's the case you should remove it altogether.